### PR TITLE
Cg 171 person edit

### PR DESCRIPTION
**Pull request by jennikate**

### CODE REVIEW

File: src/components/Forms/CreatePerson.jsx [hunk 1]
```diff
@@ -7,6 +7,9 @@ const CreatePerson = ({
   const urlParams = location.search.split('source=');
   const source = urlParams[1];
 
+  // console.log(data)
```
Comment (joshuadiablito) [inline src/components/Forms/CreatePerson.jsx]: make sure to remove logs + comments unless they are required

File: src/components/Forms/EditPerson.jsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  const personId = props.location.state.peopleId;
+  const [personData, setPersonData] = useState();
+  const [formData, setFormData] = useState();
+  const [errors, setErrors] = useState({});
+
+  // reformat dates & peopleType into individual items for form field display
+  const reformatFields = (data) => {
+    const tempObj = data;
+    const splitDoB = data.dateOfBirth.split('-');
+    const splitDocumentExpiryDate = data.documentExpiryDate.split('-');
```
Comment (joshuadiablito) [inline src/components/Forms/EditPerson.jsx]: you can use destructuring for some easy to read code:

```suggestion
    const [documentExpiryDateYear, documentExpiryDateMonth, documentExpiryDateDay] = data.documentExpiryDate.split('-');
```

File: src/components/Forms/CreatePerson.jsx (hunk 1 above)
Comment (jennikate) [reply src/components/Forms/CreatePerson.jsx]: cleaned up :)

File: src/components/Forms/EditPerson.jsx (hunk 2 above)
Comment (jennikate) [reply src/components/Forms/EditPerson.jsx]: refactored reformatFields and reformatDate

File: src/components/Forms/CreatePerson.jsx (hunk 1 above)
Comment (p3dr0migue1) [inline src/components/Forms/CreatePerson.jsx]: console.log :scream:

Comment (jennikate) [reply src/components/Forms/CreatePerson.jsx]: fixed

**Pull request merged**